docs/UI/Slider/index.md

```diff
@@ -84,25 +84,21 @@ Slider with OK Button press
 }
 ```
 
-#### Example Error
```

Comment:
Can you add titles that describe these two different success responses?

Reply:
Done in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/249/commits/f06a25f318deeb879431109a96e314c0da3d21fb